osgi/felixtest/pom.xml

```diff
@@ -20,7 +20,6 @@
     <description>Hibernate Validator OSGi integration test for Felix using Payara Server</description>
 
     <properties>
-        <hibernate.validator.version>6.0.3-SNAPSHOT</hibernate.validator.version>
```

Comment:
Ahh so maybe that's what caused those random test failures ?

Reply:
That's what I hope :). But I'm not sure. The server is complaining about a missing dependency in a microprofile so that might be it.